src/devices/bus/coco/coco_multi.cpp

```diff
@@ -185,6 +195,14 @@ MACHINE_CONFIG_START(coco_multipak_device::device_add_mconfig)
 	MCFG_COCO_CARTRIDGE_HALT_CB(DEVWRITELINE(DEVICE_SELF, coco_multipak_device, multi_slot4_halt_w))
 MACHINE_CONFIG_END
 
+INPUT_PORTS_START( coco_multipack )
+	PORT_START( "MPI Slot Switch" )
+	PORT_DIPNAME( 0x03, 0x03, "Multi-Pak Slot Switch" ) PORT_CHANGED_MEMBER(DEVICE_SELF, coco_multipak_device, switch_changed, nullptr)
```

Comment:
Is this actually a DIP switch?  If it's some other kind of configuration switch or jumpers, it should use `PORT_CONFNAME` and `PORT_CONFSETTING`.  DIP switches should be real DIP switches.